src/ImageKit/Constants/SupportedTransforms.php

```diff
@@ -67,11 +67,28 @@ class SupportedTransforms
         'overlayImageFocus' => 'oifo'
     ];
 
+    private static $ifTransforms = [
+        'height' => 'h',
+        'width' => 'w',
+        'aspectRatio' => 'ar',
+        'originalHeight' => 'ih',
+        'originalWidth' => 'iw',
+        'originalAspectRatio' => 'iar',
+    ];
+
     /**
      * @return array<string, string>
      */
     public static function get()
     {
         return self::$transforms;
     }
+    
+    /**
+     * @return array<string, string>
+     */
+    public static function getIf()
+    {
+        return self::$ifTransforms;
+    }
 }
```

Comment:
We need to start supporting `raw` in the supported list of transformations so that the user can enter arbitrary values which are not covered by SDK e.g. complex conditional transformations or video layers.
Its already being done in Ruby SDK
<img width="1029" alt="image" src="https://user-images.githubusercontent.com/45416977/173516358-6391d865-f23f-4738-9642-3a2091540cad.png">
```
If you want to generate transformations in your application and add them to the URL as it is, use the raw parameter.
```

Reply:
will se once conditional transformation is discussed

Reply:
No, I mean `raw` parameter has a specical meaning that it will put the user provided transformation string as it in the URL as shown in the Ruby SDK test case - https://github.com/imagekit-developer/imagekit-ruby/blob/master/test/imagekit/url_test.rb#L142

Reply:
done